yt_dlp/extractor/line.py

```diff
@@ -82,15 +84,50 @@ def _real_extract(self, url):
 
         # like_count requires an additional API request https://tv.line.me/api/likeit/getCount
 
+        thumbnail = url_or_none(try_get(video_info, lambda x: x['meta']['cover']['source']))
+        if thumbnail:
+            thumbnail = thumbnail.split('?')[0]
+
+        subtitles = {}
+        sub_fan = "-fan"
```

Comment:
no need to set this as a variable imo
just use directly

Reply:
That might return a variable that doesn't exist. Or are you talking about "-fan"?

Reply:
yeah
just directly include `-fan` in the fstrings

Reply:
Ok let me know if this is right https://github.com/yt-dlp/yt-dlp/pull/1927/commits/aa1ab66cf8a46501d9309ac0dbfa0a33de2ee07c

Reply:
yup looks good to me.